src/test/java/uk/ac/ebi/eva/test/utils/GenotypedVcfJobTestUtils.java

```diff
@@ -0,0 +1,196 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.zip.GZIPInputStream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static uk.ac.ebi.eva.test.utils.JobTestUtils.count;
+import static uk.ac.ebi.eva.test.utils.JobTestUtils.getLines;
+import static uk.ac.ebi.eva.test.utils.TestFileUtils.getResource;
+
+/**
+ * Utility functions to test
+ */
+public class GenotypedVcfJobTestUtils {
+
+    private static final String MOCK_VEP = "/mockvep.pl";
+
+    public static final String INPUT_VCF_ID = "1";
+
+    public static final String INPUT_STUDY_ID = "genotyped-job";
+
+    private static final String INPUT_FILE = "/input-files/vcf/genotyped.vcf.gz";
+
+    public static final String COLLECTION_FILES_NAME = "files";
+
+    public static final String COLLECTION_VARIANTS_NAME = "variants";
+
+    private static final int EXPECTED_ANNOTATIONS = 537;
+
+    private static final int EXPECTED_VARIANTS = 300;
+
+    private static final int EXPECTED_VALID_ANNOTATIONS = 536;
+
+    public static VariantDBIterator getVariantDBIterator(String dbName) throws IllegalAccessException,
+            ClassNotFoundException, InstantiationException, StorageManagerException {
+        VariantStorageManager variantStorageManager = StorageManagerFactory.getVariantStorageManager();
+        VariantDBAdaptor variantDBAdaptor = variantStorageManager.getDBAdaptor(dbName, null);
+        return variantDBAdaptor.iterator(new QueryOptions());
+    }
+
+    /**
+     * 4 annotation flow annotation input vep generate step
+     *
+     * @param vepInputFile
+     * @throws IOException
+     */
+    public static void checkAnnotationInput(File vepInputFile) throws IOException {
+        BufferedReader testReader = new BufferedReader(new InputStreamReader(new FileInputStream(
+                getResource("/expected-output/preannot.sorted"))));
+        BufferedReader actualReader = new BufferedReader(new InputStreamReader(new FileInputStream(
+                vepInputFile.toString())));
+
+        ArrayList<String> rows = new ArrayList<>();
+
+        String s;
+        while ((s = actualReader.readLine()) != null) {
+            rows.add(s);
+        }
+        Collections.sort(rows);
+
+        String testLine = testReader.readLine();
+        for (String row : rows) {
+            assertEquals(testLine, row);
+            testLine = testReader.readLine();
+        }
+        assertNull(testLine); // if both files have the same length testReader should be after the last line
+    }
+
+
+    /**
+     * Annotation load step: check documents in DB have annotation (only consequence type)
+     *
+     * @param dbName
+     * @throws IllegalAccessException
+     * @throws ClassNotFoundException
+     * @throws InstantiationException
+     * @throws StorageManagerException
```

Comment:
javadoc detects the exceptions thrown even without those annotations. If the exception names are explanatory enough to why the method throws it, I would still remove them.